### PR TITLE
docs: derek suggested polish

### PR DESCRIPTION
**Pull request by leeederek**
Light touchups

Changelog picked up from commits here:

docs: derek suggested polish

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,21 +3,21 @@ Ethereum Network Package
...
-these and other parameters are configurable through a json file. 
+By default, this package spins up a single node with a [`geth`](https://github.com/kurtosis-tech/eth-network-package/blob/main/src/el/geth/geth_launcher.star) EL client and [`lighthouse`](https://github.com/kurtosis-tech/eth-network-package/blob/main/src/cl/lighthouse/lighthouse_launcher.star) CL client and comes with [five prefunded keys](https://github.com/kurtosis-tech/eth-network-package/blob/main/src/prelaunch_data_generator/genesis_constants/genesis_constants.star) for testing, but
+these and other parameters are configurable through a json file. Read more about the [node architecture here](https://ethereum.org/en/developers/docs/nodes-and-clients/node-architecture/)
```
Comment (tedim52) [inline README.md:20]: nit: could we add a period at end of sentence?

File: README.md [hunk 2]
```diff
@@ -61,17 +61,18 @@ For example, this `eth-network-params.json` adds a second node, running a differ
 	"global_client_log_level": "info"
 }
 ```
-To run the package with your desired configuration, simply run
+To run the package with your desired configuration (as specified in your `eth-network-params.json` file, simply run:
```
Comment (tedim52) [inline README.md:64]: nit: can we we close the parenthesis? `...(as specified in your `eth-network-params.json` file), simply run:`

**Review by tedim52 — APPROVED** [review]
Great suggestions! jus small nits

File: README.md (hunk 2 above)
Comment (leeederek) [reply README.md:64]: Thanks for catching my silly mistake, will do!
```suggestion
To run the package with your desired configuration (as specified in your `eth-network-params.json` file), simply run:
```

File: README.md (hunk 1 above)
Comment (leeederek) [reply README.md:20]: Yes. Fixing now:

```suggestion
these and other parameters are configurable through a json file Read more about the [node architecture here](https://ethereum.org/en/developers/docs/nodes-and-clients/node-architecture/).
```

**Pull request merged**